### PR TITLE
More topological relation equivalences

### LINKED ISSUE #510: define more topological relation equivalences
https://docs.ogc.org/is/22-047r1/22-047r1.html#_equivalent_rcc8_egenhofer_and_simple_features_topological_relations
9.5.  Equivalent RCC8, Egenhofer and Simple Features Topological Relations
has a rich table of equivalences.

But as far as I can see, `geo.ttl` defines only these:
```ttl
:ehDisjoint owl:equivalentProperty :sfDisjoint .
:ehEquals owl:equivalentProperty :sfEquals .
:ehMeet	owl:equivalentProperty :sfTouches .
```

Please add the following:
- `sfOverlaps equivalentProperty ehOverlap`
- Equivalences of `rcc8` relations 
- `subPropertyOf` for the `+` combinations, eg 
`ehContains subPropertyOf sfContains. ehCovers subPropertyOf sfContains`
- `complementOf` for the negations `¬`, eg
`sfIntersects owl:complementOf ehDisjoint`
and thus 
`sfIntersects owl:complementOf sfDisjoint`

In addition, change the table to use the actual prop names (eg `geo:sfDisjoint` in the **Simple Features** column), and link to their definitions.

### PR DESCRIPTION
**Pull request by situx**
Closes #510

Adds topological relations missing from geo.ttl and updates the specification with better links in the topological relation equivalences table.

@VladimirAlexiev

### CODE REVIEW

**Review by nicholascar — APPROVED** [review]

**Review by nicholascar — APPROVED** [review]
I think the subPropertyOfs are correct

File: vocabularies/geo.ttl [hunk 1]
```diff
@@ -261,9 +265,10 @@ geo:ehCovers
 
 geo:ehDisjoint
 	a rdf:Property, owl:ObjectProperty ;
-	owl:equivalentProperty :sfDisjoint ;
-	rdfs:domain geo:SpatialObject ;
-	rdfs:range geo:SpatialObject ;
+	owl:equivalentProperty :sfDisjoint, :rcc8dc ;
+	rdfs:domain :SpatialObject ;
+	rdfs:range :SpatialObject ;
+	owl:complementOf :sfIntersects ;
```
Comment (mperry455) [inline vocabularies/geo.ttl:271]: Isn't owl:complementOf more of a class axiom? Perhaps owl:propertyDisjointWith would be better here.

File: vocabularies/geo.ttl [hunk 2]
```diff
@@ -594,9 +609,10 @@ geo:sfCrosses
 
 geo:sfDisjoint
 	a rdf:Property, owl:ObjectProperty ;
-	owl:equivalentProperty :ehDisjoint ;
-	rdfs:domain geo:SpatialObject ;
-	rdfs:range geo:SpatialObject ;
+	owl:equivalentProperty :ehDisjoint, :rcc8dc ;
+	rdfs:domain :SpatialObject ;
+	rdfs:range :SpatialObject ;
+	owl:complementOf :sfIntersects ;
```
Comment (mperry455) [inline vocabularies/geo.ttl:615]: owl:propertyDisjointWith instead of owl:complementOf?

File: vocabularies/geo.ttl [hunk 3]
```diff
@@ -622,8 +638,9 @@ geo:sfEquals
 
 geo:sfIntersects
 	a rdf:Property, owl:ObjectProperty ;
-	rdfs:domain geo:SpatialObject ;
-	rdfs:range geo:SpatialObject ;
+	rdfs:domain :SpatialObject ;
+	rdfs:range :SpatialObject ;
+	owl:complementOf :sfDisjoint, :ehDisjoint ;
```
Comment (mperry455) [inline vocabularies/geo.ttl:643]: owl:propertyDisjointWith instead of owl:complementOf?

**Comment by situx** [comment]
Meeting 7th August 2024: We would like to go with owl:propertyDisjointWith

**Comment by situx** [comment]
I changed owl:complementOf to owl:propertyDisjointWith. With that I will now merged the pull request

**Pull request merged**